### ISSUE TITLE
WDE does not restore video mode

### ISSUE THREAD

**Issue by boeckmann** [Baron-von-Riedesel/WDe#2]
After leaving WDE the video mode stays at 43? screen lines. Would be convinient if it restores the video mode which was active when starting WDE.

**Comment by Baron-von-Riedesel** [comment]
Yes, I'm aware of this little inconvenience, but I think that's part of the charm of DOS apps.
